submissions/kovalov/memory-pair-game/main.js

```diff
@@ -0,0 +1,137 @@
+const imageNames = [
+  'img-1.png',
+  'img-2.png',
+  'img-3.png',
+  'img-4.png',
+  'img-5.png',
+  'img-6.png',
+  'img-7.png',
+  'img-8.png',
+];
+
+let currentCard = '';
+let previousCard = '';
+let isActive = true;
+let timesClicked = 0;
+let timesMatched = 0;
+
+const gameContainer = document.querySelector('[data-game]');
```

Comment:
Better to use `getElementById` for search because of performance reasons, `getElementById` is more specific. Article to see how you can use it
https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes